Simplified/NYPLBookDetailsProblemDocumentViewController.swift

```diff
@@ -171,6 +172,11 @@
   }
   
   func submitButtonWasPressed() {
+    guard let supportEmail = AccountsManager.shared.currentAccount?.supportEmail else {
```

Comment:
did we get confirmation that we should not use a default email anymore -- that gethelp@nypl.org for example, or another one Risa can provide maybe? For support purposes it would be good not to silently fail here.

Reply:
This is supposed to be a highly exceptional case and Risa did not want to send non-NYPL users to that email address. We the submit button is supposed to be disabled if there is no support email.